ml-proto/host/parser.mly

```diff
@@ -235,15 +229,17 @@ expr1 :
   | BR var expr_opt { fun c -> Br ($2 c label, $3 c) }
   | BR_IF var expr { fun c -> Br_if ($2 c label, None, $3 c) }
   | BR_IF var expr expr { fun c -> Br_if ($2 c label, Some ($3 c), $4 c) }
+  | BR_TABLE var var_list expr
+    { fun c -> let xs, x = Lib.List.split_last ($2 c label :: $3 c label) in
+      Br_table (xs, x, None, $4 c) }
```

Comment:
LR(1) generators like Yacc cannot deal with `var_list var` easily, will cause shift/reduce conflicts. So annoyingly, you have to turn it around.